Modules/Microsoft365DSC/DSCResources/MSFT_AADAuthenticationMethodPolicyX509/MSFT_AADAuthenticationMethodPolicyX509.schema.mof

```diff
@@ -0,0 +1,50 @@
+[ClassVersion("1.0.0")]
+class MSFT_MicrosoftGraphX509CertificateAuthenticationModeConfiguration
+{
+    [Write, Description("Rules are configured in addition to the authentication mode to bind a specific x509CertificateRuleType to an x509CertificateAuthenticationMode. For example, bind the policyOID with identifier 1.32.132.343 to x509CertificateMultiFactor authentication mode."), EmbeddedInstance("MSFT_MicrosoftGraphX509CertificateRule")] String Rules[];
+    [Write, Description("The type of strong authentication mode. The possible values are: x509CertificateSingleFactor, x509CertificateMultiFactor, unknownFutureValue."), ValueMap{"x509CertificateSingleFactor","x509CertificateMultiFactor","unknownFutureValue"}, Values{"x509CertificateSingleFactor","x509CertificateMultiFactor","unknownFutureValue"}] String X509CertificateAuthenticationDefaultMode;
+};
+[ClassVersion("1.0.0")]
+class MSFT_MicrosoftGraphX509CertificateRule
+{
+    [Write, Description("The identifier of the X.509 certificate. Required.")] String Identifier;
+    [Write, Description("The type of strong authentication mode. The possible values are: x509CertificateSingleFactor, x509CertificateMultiFactor, unknownFutureValue. Required."), ValueMap{"x509CertificateSingleFactor","x509CertificateMultiFactor","unknownFutureValue"}, Values{"x509CertificateSingleFactor","x509CertificateMultiFactor","unknownFutureValue"}] String X509CertificateAuthenticationMode;
+    [Write, Description("The type of the X.509 certificate mode configuration rule. The possible values are: issuerSubject, policyOID, unknownFutureValue. Required."), ValueMap{"issuerSubject","policyOID","unknownFutureValue"}, Values{"issuerSubject","policyOID","unknownFutureValue"}] String X509CertificateRuleType;
+};
+[ClassVersion("1.0.0")]
+class MSFT_MicrosoftGraphX509CertificateUserBinding
+{
+    [Write, Description("The priority of the binding. Azure AD uses the binding with the highest priority. This value must be a non-negative integer and unique in the collection of objects in the certificateUserBindings property of an x509CertificateAuthenticationMethodConfiguration object. Required")] UInt32 Priority;
+    [Write, Description("Defines the Azure AD user property of the user object to use for the binding. The possible values are: userPrincipalName, onPremisesUserPrincipalName, email. Required.")] String UserProperty;
+    [Write, Description("The field on the X.509 certificate to use for the binding. The possible values are: PrincipalName, RFC822Name.")] String X509CertificateField;
+};
+[ClassVersion("1.0.0")]
+class MSFT_MicrosoftGraphExcludeTarget2
```

Comment:
To avoid conflicts across resources in the future, we took the decision to always rename these CIMInstances as:

MSFT_resourceNameCIMInstance

Please review every schema.mof and module files in this Pull Request to adhere to this naming standard. Thanks